src/SFA.DAS.LevyTransferMatching.Api/Models/AccountDto.cs

```diff
@@ -4,13 +4,15 @@ namespace SFA.DAS.LevyTransferMatching.Api.Models
 {
     public class AccountDto
     {
+        public string EncodedAccountId { get; set; }
         public string DasAccountName { get; set; }
-        public decimal RemainingTransferAllowance { get; set; }
+        public decimal? RemainingTransferAllowance { get; set; }
```

Comment:
As on the web PR, it's not obvious why this has to change.

Reply:
Agreed to review DTOs and to discuss request/response convention as part of a team, and to insert tech debt backlog to address this.

See: https://github.com/SkillsFundingAgency/das-levy-transfer-matching-web/pull/29#discussion_r662948503

Reply:
As per comment on web PR, incurring tech debt to deal with some of this is fine, but perhaps not for this Dto, which we'd only have to split/change back later.

Reply:
I've split this out as requested, into `Account` and `UserAccount` entities, and associated DTOs.